src/main/java/com/actiontech/dble/backend/mysql/nio/handler/MultiNodeLoadDataHandler.java

```diff
@@ -712,10 +712,10 @@ void handleEndPacket(MySQLPacket curPacket, AutoTxOperation txOperation, boolean
             if (txOperation == AutoTxOperation.COMMIT) {
                 session.checkBackupStatus();
                 session.setBeginCommitTime();
-                handler.commit();
+                handler.commit(null);
```

Comment:
add a new interface method for impcilit commit

Reply:
ok